cfgov/research_reports/models.py

```diff
@@ -47,6 +51,10 @@ def get_researchers():
     ])
 
 
+def _get_deploy_environment():
+    return getattr(settings, 'DEPLOY_ENVIRONMENT', None)
```

Comment:
```suggestion
    return getattr(settings, 'DEPLOY_ENVIRONMENT', 'local')
```

Reply:
`DEPLOY_ENVIRONMENT` isn't set in local docker, we can use the above to provide local when it's unset